src/components/oversikt-krav/KravSammendrag.tsx

```diff
@@ -13,18 +13,22 @@ import OversiktKravItem from '../../state/oversikt-krav/OversiktKravItem';
 import formaterIsoTimestampAsNoTime from '../../utils/formatIsoTimestampAsNoTimestamp';
 import Lenke from 'nav-frontend-lenker';
 import slettRefusjonskrav from '../../api/slettRefusjonskrav/slettRefusjonskrav';
+import SlettetKravKvittering from '../slettetKravKvittering';
+import estimertRefusjon from '../../utils/estimertRefusjon';
 
 interface KravSammendragProps {
-  items?: OversiktKravItem[];
+  items: OversiktKravItem[];
   innsending?: string;
   dispatch?: any;
 }
 
 const KravSammendrag = (props: KravSammendragProps) => {
+  const [slettetKrav, setSlettetKrav] = useState<OversiktKravItem | undefined>(undefined);
```

Comment:
handleSlettInnsending
slettRefusjonskrav
slettetKrav

Synes vi burde ha felles substantiv på disse for å øke konsistens og lesbarhet.

Reply:
Kunne kanskje ha brukt kun Krav?

Reply:
Enig, renamer